ads/inabox/inabox-messaging-host.js

```diff
@@ -129,10 +132,9 @@ export class InaboxMessagingHost {
       return false;
     }
 
-    const allowedTypes = adFrame.iframe.dataset['ampAllowed'];
-    // having no whitelist is legacy behavior so assume all types are allowed
-    if (allowedTypes &&
-        allowedTypes.split(/\s*,\s*/).indexOf(request['type']) === -1) {
+    const allowedTypes = adFrame.iframe.dataset['ampAllowed'] || '';
+    if (allowedTypes.split(/\s*,\s*/).concat(READ_ONLY_MESSAGES)
```

Comment:
Can we gate this behind an experiment flag, and then run a server-driven experiment?